DataProcessor/DataUnpacker.h

```diff
@@ -113,10 +121,13 @@ public slots:
     bool mcu_check=false;
     bool imd_status=false;
     bool bps_fault=false;
-    bool bms_canbus_failure=false, voltage_failsafe=false, current_failsafe=false, supply_power_failsafe=false, memory_failsafe=false, relay_failsafe=false;
-    bool mps_enable=true, mppt_contactor=false, low_contactor=false, motor_controller_contactor=false;
+    bool bms_can_heartbeat=false, voltage_failsafe=false, current_failsafe=false, input_power_supply_failsafe=false, relay_failsafe=false;
+    bool discharge_enable=true, mppt_contactor=false, low_contactor=false, motor_controller_contactor=false;
     bool restart_enable=false;
     QVector<float> cell_group_voltages;
+    bool cell_balancing_active = false, mcu_hv_en = false, charge_interlock_failsafe = false, thermistor_b_value_table_invalid = false;
+    bool mcu_stat_fdbk = false, dcdc_valid = false, supplemental_valid=false, charge_enable = false;
+    int mc_status = 0;
```

Comment:
```suggestion
    bool discharge_enable=false, charge_enable=false, bms_can_heartbeat=false;
    bool mcu_hv_en=false, mcu_stat_fdbk=false, dcdc_valid=false, supplemental_valid=false, mppt_contactor=false, low_contactor=false, motor_controller_contactor=false;
    bool voltage_failsafe=false, current_failsafe=false, relay_failsafe=false, cell_balancing_active=true, charge_interlock_failsafe=false, thermistor_b_value_table_invalid=false, input_power_supply_failsafe=false;
    QVector<float> cell_group_voltages;
    bool restart_enable=true;
    int mc_status=0;
```
- **First, `restart_enable` should be initialized to true to fit the startup/shutdown flowchart, `discharge_enable` should be initialized to false since that's its non-error state, and `cell_balancing_active` should be initialized to true since that's its non-error state.**
- Again, a bit nit-picky, but could you please reorganize the variables so that they are grouped by similar signals?
- Also, to keep it consistent with the other variables, please remove the spaces around the equals sign in `<variable>=<value>`.